src/__test__/components/data-management/SamplesTable.test.jsx

```diff
@@ -115,71 +110,67 @@ describe('Samples table', () => {
     await storeState.dispatch(loadEnvironment('test'));
   });
 
-  it('Shows option to download datasets if there are no samples', async () => {
+  it('Download datasets works correctly and shows up if there are no samples', async () => {
```

Comment:
Change to `Clone projects` instead of `Download datasets`?

Reply:
This test suite seeme pretty big. Would it make sense to split it up into 2 parts? Test that the example experiments are shown and another test to check if the download mechanism works?